File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Clouds.java
package com.riders.thelab.data.remote.dto.weather;

import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Clouds {

    @Json(name = "all")
    int cloudiness;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Clouds.kt
package com.riders.thelab.data.remote.dto.weather

import kotlinx.serialization.SerialName
import java.io.Serializable

@kotlinx.serialization.Serializable
data class Clouds constructor(
    @SerialName("all")
    val cloudiness: Int = 0
) : Serializable
